pandas/tests/io/formats/test_format.py

```diff
@@ -1417,6 +1417,18 @@ def test_to_string_float_index(self):
                     '5.0  4')
         assert result == expected
 
+    def test_to_string_complex_float_formatting(self):
+        fmt.set_option('display.precision', 5)
```

Comment:
do we need to use the context manger here?

Reply:
yep

Reply:
can you use the context manager here